cli/ops/worker_host.rs

```diff
@@ -42,9 +43,19 @@ fn create_web_worker(
   permissions: Permissions,
   specifier: ModuleSpecifier,
   has_deno_namespace: bool,
+  import_map_path: Option<String>,
 ) -> Result<WebWorker, ErrBox> {
-  let state =
-    State::new_for_worker(global_state, Some(permissions), specifier)?;
+  let maybe_import_map = match import_map_path.as_ref() {
+    None => None,
+    Some(file_path) => Some(ImportMap::load(file_path)?),
+  };
```

Comment:
Since it's a runtime feature then there should be permission check for import map (`state.check_read(import_map_path)`)